### ISSUE TITLE
Exception with simple usage

### ISSUE THREAD

**Issue by CameronNemo** [eht16/python-logstash-async#42]
```py
import logging

from logstash_async.handler import AsynchronousLogstashHandler
from logstash_async.transport import BeatsTransport

lgr = logging.getLogger()
lgr.addHandler(AsynchronousLogstashHandler('10.222.0.20', 5056, database_path=None, transport=BeatsTransport))
lgr.error('fnord')
```

```
2019-07-03 08:55:45: exception: An error occurred while sending events: send() missing 1 required positional argument: 'events'
Traceback (most recent call last):
  File "/Users/cameronnemo/docs/elk/py-logstash-demo/venv/lib/python3.7/site-packages/logstash_async/worker.py", line 118, in _fetch_events
    self._fetch_event()
  File "/Users/cameronnemo/docs/elk/py-logstash-demo/venv/lib/python3.7/site-packages/logstash_async/worker.py", line 140, in _fetch_event
    self._event = self._queue.get(block=False)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/queue.py", line 167, in get
    raise Empty
_queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/cameronnemo/docs/elk/py-logstash-demo/venv/lib/python3.7/site-packages/logstash_async/worker.py", line 224, in _flush_queued_events
    self._send_events(events)
  File "/Users/cameronnemo/docs/elk/py-logstash-demo/venv/lib/python3.7/site-packages/logstash_async/worker.py", line 255, in _send_events
    self._transport.send(events, use_logging=use_logging)
TypeError: send() missing 1 required positional argument: 'events'

2019-07-03 08:55:45: error: Error on closing transport: close() missing 1 required positional argument: 'self'
```

**Comment by eht16** [comment]
Thanks for reporting.
Fixed in 1.5.1.

**Comment by CameronNemo** [comment]
Wow that was fast! Thank you.
